src/main/java/org/mastodon/mamut/launcher/RecentProjectsPanel.java

```diff
@@ -140,4 +154,44 @@ private void remakeGUI( final Consumer< String > projectOpener )
 		revalidate();
 		repaint();
 	}
+
+
+	public static final class MouseDblClickOpenPath implements MouseListener {
+		final String Uri;
+		public MouseDblClickOpenPath(final String pathOnDblClick) {
+			Uri = Paths.get(pathOnDblClick).getParent().toUri().toString();
+		}
+
+		@Override
+		public void mouseClicked(MouseEvent mouseEvent) {
+			if (mouseEvent.getClickCount() == 2) openUrl( Uri );
```

Comment:
action no. 3: here is the click processing, actually must be a dbl-click
it opens the folder at the memorized path (the parent folder of the dlg item content)